freeglut/freeglut/src/fg_internal.h

```diff
@@ -572,9 +670,10 @@ do                                            \
 {                                             \
```

Comment:
Maybe we should bring this up on the forum, separate from your current work. Indeed any modern compiler should optimize this difference away with ease.